core/server/models/offer.js

```diff
@@ -0,0 +1,53 @@
+const ghostBookshelf = require('./base');
+
+const Offer = ghostBookshelf.Model.extend({
+    tableName: 'offers',
+
+    async onSaving(model, _attr, options) {
+        ghostBookshelf.Model.prototype.onSaving.apply(this, arguments);
+
+        if (model.get('name')) {
```

Comment:
I think this can be handled by the Offers module so all the logic is in one place - wdyt?

Reply:
Comment below ⬇️

Reply:
Done